### DISCUSSION TITLE
PovertyMap Domains

### DISCUSSION THREAD

**Discussion by niki-amini-naieni** [p-lambda/wilds#105] [Q&A] [answered]
Hello,

I am wondering why in appendix E.8 of the original WILDS paper, the authors state "the domain d represents the country the image was taken in and whether the image is of an urban or rural area" (E.8.1, p. 102), but for Group DRO, the authors use only the country as the domain. Why did they not use a combination of the country and the urban/rural subpopulation as the domain? In the former case, there would be 23 domains (1 for each country). In the latter case, there would be 46 domains (1 for each country and urban/rural combination). 

Link to paper reference: https://arxiv.org/pdf/2012.07421.pdf

Thank you!

**Comment by kohpangwei** [comment, marked as answer]
Hi Niki! I don't think there was a particular reason -- for IRM and CORAL we used countries as the domain, so we decided to be consistent with Group DRO. For IRA and CORAL, it seemed more reasonable to assume that the representations should be invariant over countries instead of over urban/rural areas, but this was a heuristic. IIRC, we also tried running these algorithms with the 23 x 2 = 46 domains but performance was roughly similar (i.e., comparable to or worse than ERM). Thanks for asking.

**Reply by niki-amini-naieni** [reply]
Thank you!
